editions/tw5.com/tiddlers/releasenotes/5.4.0/#9333.tid

```diff
@@ -0,0 +1,48 @@
+title: $:/changenotes/5.4.0/#9333
+description: Intergrate Tiddlywiki palette colors and settings to custom CSS properties
+release: 5.4.0
+tags: $:/tags/ChangeNote
+change-type: feature
+change-category: developer
+github-links: https://github.com/TiddlyWiki/TiddlyWiki5/pull/9333
+github-contributors: Leilei332
+
```

Comment:
hmm, Since there is a second documentation tiddler, which shows both variants, I think the release note can only show the new behaviour - Just a thought